### PR TITLE
docs: improve discoverability of how to edit this file

### PR DESCRIPTION
**Pull request by jsoref**
Give people a bit of a chance...

### CODE REVIEW

**Comment by bcoe** [comment]
I am waiting to merge this until I can create the PRs en-masse.

**Comment by bcoe** [comment]
@jsoref perhaps we can find a place to actually link to the baseline README? Perhaps in the contributing section we could add notes about some of the files being auto-generated, with links, e.g., here:

https://github.com/googleapis/nodejs-storage#contributing

**Comment by jsoref** [comment]
@bcoe: thinking about it a number of times... yes. That section should at the very least clearly indicate that the repository is significantly a generated repository.

I'm not sure I read that section (if I hadn't, that'd be my fault).

---
(What follows is probably more applicable to the other PR, but ...)
In following the links, it actually only says that the PR itself should be titled per conventional commits and not the commit message, which is confusing. -- And yes, I remember you saying elsewhere it's a guideline and not mandatory.

If the PR title itself should follow conventional, that should really be in the PR template.

My default thought is that the PR template is the right place for these warnings... 

So, playing around using this very artificial url:
https://github.com/jsoref/synthtool/compare/master...googleapis:master
I get:
ⓘ Remember, contributions to this repository should follow its [contributing guidelines](https://github.com/jsoref/synthtool/blob/master/.github/CONTRIBUTING.md) and [code of conduct](https://github.com/jsoref/synthtool/blob/master/CODE_OF_CONDUCT.md).

Of those items, the one that should have some of this is CONTRIBUTING.md, which doesn't currently have anything helpful.

**Comment by bcoe** [comment]
> It actually only says that the PR itself should be titled per conventional commits and not the commit message, which is confusing. 

This is intentional, as we've been working on automation that carries the commit title over when merging, rather than the commit body; So I think we'll be able to move towards linting just the title, not the commit messages (which are more of a hassle to edit).

> I'm not sure I read that section (if I hadn't, that'd be my fault).

Rather than landing this comment, which I doubt folks will find, let's just go ahead and add a section along the lines of:

```
Please note that `README.md`, `samples/README.md`, and a variety of configuration files in this repository (`.nycrc`, `tsconfig.json`) are generated from a central template. If you would like to make edits, check out the template [here](https://github.com/googleapis/synthtool/tree/master/synthtool/gcp/templates/node_library).
```

Would you like to take shot at this copy? I'm figuring I can merge all these updates at once; including the apostrophe fix, in one mass PR once we're happy.

**Comment by jsoref** [comment]
Ok. It feels like you're imaging a `piper` with "please check out" ... lemme see what I can do... I almost have it...

**Comment by jsoref** [comment]
@bcoe: how's this look?

File: synthtool/gcp/templates/node_library/README.md [hunk 1]
```diff
@@ -120,6 +120,13 @@ More Information: [Google Cloud Platform Launch Stages][launch_stages]
 
 Contributions welcome! See the [Contributing Guide](https://github.com/{{ metadata['repo']['repo'] }}/blob/master/CONTRIBUTING.md).
 
+Please note that this [README.md](README.md), the [samples/README.md](samples/README.md),
+and a variety of configuration files in this repository (including `.nycrc` and `tsconfig.json`)
+are generated from a central template.
+If you would like to make edits to one of those files
```
Comment (bcoe) [inline synthtool/gcp/templates/node_library/README.md]: nit: I might make `If you would like to make edits to one of those files` start in a new paragraph, just to break up the block of text a bit.

But non blocker; totally up to you.

**Comment by bcoe** [comment]
left one `nit` but this is looking good to go from my perspective, I'm going to wait and land with a few more documentation updates so that I can send out the PR en masse.

**Comment by jsoref** [comment]
@bcoe: that's reasonable (and partially why I split the text into multiple lines, it made such a change theoretically easier w/o blame shifting).

**Pull request merged**